CONTRIBUTING.md

```diff
@@ -9,14 +9,44 @@ If you'd like to [run the test suite](#running-tests), you will also need `grype
 
 If you'd like to [generate an SBOM](#generating-an-sbom), you will also need `syft` downloaded and in your `PATH`.
 
-### Building the image
+### Building the images
 
-To build the docker image, run:
+To build all of the docker images, run:
 
 ```bash
 pipenv run invoke build
 ```
 
+If you only want to build a specific image, you can pass in a tool and/or an environment. For example:
+
+```bash
+pipenv run invoke build --tool=terraform --environment=none
+```
+
+To see the list of possible tools, run the following command. If you don't specify a tool, `build` will assume that you want to build "all" of the
+tools.
+
+```bash
+pipenv run python3 -c \
```

Comment:
I just didn't think `pipenv run python3 -c "from easy_infra import constants; print(list(constants.TOOLS))"` was that big of a deal, especially given it isn't something someone familiar with the project would need to do, it's more of a one-time getting familiar with the project or periodically checking in to see what new tools are available kind of thing.  I really don't have a good justification otherwise lol